.travis.yml

```diff
@@ -14,16 +14,14 @@ env:
     - DOCKER_BUILDKIT=1
     - DOCKER_MACHINE_ARM64_NAME=travis-ci-kong-build-tools-${TRAVIS_JOB_ID}
   matrix:
-    - CACHE=true BUILDX=true KONG_SOURCE=master
-    - CACHE=false BUILDX=true KONG_SOURCE=master
     - RESTY_IMAGE_BASE=src KONG_SOURCE=master
     - PACKAGE_TYPE=rpm RESTY_IMAGE_BASE=rhel RESTY_IMAGE_TAG=6 KONG_SOURCE=master
     - PACKAGE_TYPE=rpm RESTY_IMAGE_BASE=rhel RESTY_IMAGE_TAG=7 KONG_SOURCE=master
     - PACKAGE_TYPE=deb RESTY_IMAGE_BASE=ubuntu RESTY_IMAGE_TAG=devel KONG_SOURCE=master
     - PACKAGE_TYPE=deb RESTY_IMAGE_BASE=ubuntu RESTY_IMAGE_TAG=rolling KONG_SOURCE=master
     - PACKAGE_TYPE=deb RESTY_IMAGE_BASE=ubuntu RESTY_IMAGE_TAG=latest KONG_SOURCE=master
     - PACKAGE_TYPE=deb RESTY_IMAGE_BASE=ubuntu RESTY_IMAGE_TAG=trusty KONG_SOURCE=master
-    - PACKAGE_TYPE=deb RESTY_IMAGE_BASE=ubuntu RESTY_IMAGE_TAG=xenial KONG_SOURCE=master
+    - PACKAGE_TYPE=deb RESTY_IMAGE_BASE=ubuntu RESTY_IMAGE_TAG=xenial KONG_SOURCE=master CACHE=false
```

Comment:
for the time being don't rely on the docker cache as it won't have arm base images yet